meta-resin-common/recipes-connectivity/networkmanager/resin-files/99dhcp_ntp

```diff
@@ -7,10 +7,13 @@ action=$2
 
 case "$action" in
 	dhcp4-change|up|connectivity-change)
+	# Run resin-ntp-config to update chronyc from ntp servers in config.json
+	/usr/bin/resin-ntp-config
```

Comment:
Do we actually need this?

Reply:
I'd leave it hanging just in case. Doesn't hurt. Chrony just returns a message saying server already present.
e.g. if something is buggered with vpn but internet is fine, we'd like chrony to do its job ok.

Reply:
Could we at least run it when connectivity-change went online? It feels that we run this excessively.